### ISSUE TITLE
Ansible 101 Live Stream Episode 11 Agenda - June 3

### ISSUE THREAD

**Issue by geerlingguy** [geerlingguy/ansible-for-devops#275]
Previous episode agenda: #272

Video link: https://www.youtube.com/watch?v=_rDzMYp-fBs

  - [x] Thank you SO MUCH to [GitHub Sponsors](https://github.com/sponsors/geerlingguy) and [Patreon Patrons](https://www.patreon.com/geerlingguy):
    - GitHub: @ludo237, @BSalita, @TDarkShadow, @royolsen, @rangerdude, @voioo, @colinodell, @MichalAugustyn, @pyguy, @rohankrishnadev, @hansbickhofe
    - Patreon: Mark Caudill, Roy Olsen, Scwan, Vasiliy K, Sebastian P
  - [x] What happened [at the end of last week's episode](#issuecomment-634784533)?
  - [x]  Questions and notes from last week's episode:
    - Christian Kemper: "​any reason you still offer paperbacks?"
    - Aaron Kulbe: "​What is the .vcl extension?"
    - Alexandre Alonso: ​"How to learn varnish?"
    - CruelGangsta: ​"Is it record?" Yes! And here's a full [Ansible 101 YouTube Playlist](https://www.youtube.com/playlist?list=PL2_OBreMn7FqZkvMYt6ATmgC0KAGGJNAN)
    - Aaron Kulbe: "​I thought collections weren't coming until version 2.10?"
    - Dan Quaye: "​Could you have more than one node using AWX? so you could use those resources to carry out jobs?"
  - [x] Ansible Inventories - [Dynamic inventory](https://docs.ansible.com/ansible/latest/user_guide/intro_dynamic_inventory.html)
    - [x] `ansible-inventory` CLI
    - [x] PHP dynamic inventory script
    - [x] Python dynamic inventory script
    - [x] [Inventory plugins](https://docs.ansible.com/ansible/latest/plugins/inventory.html) vs [Inventory scripts](https://docs.ansible.com/ansible/devel/user_guide/intro_dynamic_inventory.html#inventory-script-example-aws-ec2)
    - [x] [AWS EC2](https://docs.ansible.com/ansible/latest/plugins/inventory/aws_ec2.html) dynamic inventory plugin

**Comment by geerlingguy** [comment]
![Template-no-Credential](https://user-images.githubusercontent.com/481677/83047039-7075aa80-a00d-11ea-9448-bf3318ef0689.png)

Of _course_ it was something silly. It's always something silly when I blow something up on the live stream... Last week (May 27) I tried running the AWX Job Template that I set up, but it kept getting SSH pubkey permission denied errors.

Because I forgot to attach the SSH key I had set up to the job. Oops.
